brave/brave-core
# Add a context menu to hide "All bookmarks button"

Resolves https://github.com/brave/brave-browser/issues/6043

Demo:

https://github.com/brave/brave-core/assets/5474642/ec39746a-1d2f-4470-ae5d-8ad6fa16fbd0



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Automated - unittest & browsertest
* Manual
  * Add a bookmark to "Other bookmarks" node and see if "All bookmarks" button shows up on the bookmarks bar
  * Right click the "All Bookmarks" button and uncheck "Show All bookmarks button"
  * See if the "All bookmarks" button is hidden
  * Vice-versa.
